velox/docs/configs.rst

```diff
@@ -536,7 +536,17 @@ Each query can override the config by setting corresponding query session proper
      - integer
      -
      - Maximum concurrent TCP connections for a single http client.
-
+   * - hive.s3.max-attempts
+     - integer
+     -
+     - Maximum attempts for connections to a single http client, work together with retry-mode. By default, it's 3 for standard/adaptive mode
```

Comment:
As this document describe, default 4 for legacy mode 
```
A default value of 4 for maximum retry attempts,
```
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-retries.html

Reply:
<img width="363" alt="image" src="https://github.com/facebookincubator/velox/assets/11849056/3ea7a3ad-6c36-4875-a812-806263c6d4f3">

Get this number from code, maybe more reliable?

Reply:
their cpp_sdk and document have some difference. Let's follow sdk

Reply:
Maybe different AWS version has different default value